packages/subgraph/schema.graphql

```diff
@@ -210,7 +231,7 @@ enum VotingMode {
 type TokenVotingPlugin implements IPlugin @entity {
   "TODO: attributes should be appended to Plugins once plugin is seperated from core"
   id: ID!
-  daos: [DaoPlugin!] @derivedFrom(field: "plugin")
+  # daos: [DaoPlugin!] @derivedFrom(field: "plugin")
```

Comment:
if u missing this attribute from `IPlugin`, i guess it will fail to deploy the subgraph, have u tested deploying the subgraph?